bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BufferedChannel.java

```diff
@@ -135,6 +135,7 @@ public void write(ByteBuf src) throws IOException {
                 unpersistedBytes.addAndGet(copied);
                 if (unpersistedBytes.get() >= unpersistedBytesBound) {
                     flush();
+                    forceWrite(false);
```

Comment:
Why flush the PageCache in the synchronized code block?

Reply:
Prevents the final write data from being too large, exceeding the configured size of flushEntrylogBytes. @hangc0276